<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QWidget" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter</string>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="faster_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Faster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slower_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Slower</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_period">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Period</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdPeriod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
